src/posts/how-to-use-JAWS-on-Windows

```diff
@@ -0,0 +1,47 @@
+---
+title: HOWTO: Use JAWS on Windows
+description: "Step-by-step guide on effectively using JAWS, a screen reader software, to navigate Windows, read documents, browse the web, and explore advanced features for individuals with visual impairments."
+category: How-to
+author: Ilknur Eren
+date: 2022-06-04
+tags:
+  - howto
+---
+
+
+## Introduction
+JAWS, Job Access With Speech, is a screen reader for Windows operating system users. JAWS assists individuals with visual impairments with interacting with computers by helping them navigate applications, read documents and browse the web on Windows operating systems. 
```

Comment:
Might be good to mention that JAWS is still the most popular screenreader in the market as of 2023, you could cite WebAIM data here. https://webaim.org/projects/screenreadersurvey9/#primary